deploy/03-rbac.yaml

```diff
@@ -10,6 +10,10 @@ rules:
   - "*"
   verbs:
   - "*"
+- apiGroups:
+  - "operatorstatus.openshift.io"
+  resoruces:
+    - clusteroperators
```

Comment:
indentation level is wrong (which makes this invalid yaml)

Reply:
also you must specify a verbs stanza also.

Reply:
here's a full version of the corrected file:
https://gist.github.com/bparees/8d671e02adfa20c741dbfc30145ab6fc

Reply:
updated with gist contents and pushed ... thanks